src/app/api/storage/shoppingList.ts

```diff
@@ -0,0 +1,63 @@
+import type { ShoppingListItemType } from '../../components/ShoppingListItem';
+
+const LOCAL_STORAGE_KEY = 'shopping-list';
+
+const getAllShoppingListItemsFromStorage = (): ShoppingListItemType[] => {
```

Comment:
Is there a reason why you use a const instead of a function declaration here (and below)?

Reply:
no particular reason, just prefer it that way if it doesn't impact performance.